projects/fsastorefrontlib/src/core/checkout/store/reducers/index.ts

```diff
@@ -32,6 +32,7 @@ export function clearCheckoutState(
       action.type === CheckoutActions.PLACE_ORDER ||
       action.type === fromCartAction.START_BUNDLE
     ) {
+      localStorage.removeItem('spartacus⚿⚿fscheckout');
```

Comment:
I think this is not the way that Spartacus intended this feature to be implemented. I know that we have one more feature implemented like this, but judging from their [documentation](https://sap.github.io/spartacus-docs/state-persistence/#page-title) this is the wrong approach. Perhaps in `onRead` check if user is logged out and then set undefined to this storage key instead of just deleting it.

Reply:
Perhaps I can try to implement that in the persistence service, but not in `onRead`, because it is responsible (as the name indicates) for reading from the storage, not writing.

Reply:
Ok, try it, but as far as I saw in their documentation they are dispatching actions in `onRead` as well.